cmd/describe/cluster/cmd.go

```diff
@@ -148,31 +148,25 @@ func run(_ *cobra.Command, argv []string) {
 			phase = "(Install is taking longer than expected)"
 		}
 	}
-
+	str := "Name:                     " + cluster.Name() + "\n" +
+		"ID:                       " + cluster.ID() + "\n" +
+		"External ID:              " + cluster.ExternalID() + "\n" +
+		"AWS Account:              " + creatorARN.AccountID + "\n" +
+		"API URL:                  " + cluster.API().URL() + "\n" +
+		"Console URL:              " + cluster.Console().URL() + "\n" +
+		"Nodes:                    " + "Master: " + fmt.Sprintf("%d", cluster.Nodes().Master()) +
+		"Infra: " + fmt.Sprintf("%d", cluster.Nodes().Infra()) +
+		"Compute: " + fmt.Sprintf("%d", cluster.Nodes().Compute()) + "\n" +
+		"Region:                   " + cluster.Region().ID() + "\n" +
+		"State:                    " + string(cluster.State()) + phase + "\n" +
+		"Channel Group:            " + cluster.Version().ChannelGroup() + "\n" +
+		"Created:                  " + cluster.CreationTimestamp().Format("Jan _2 2006 15:04:05 MST") + "\n"
```

Comment:
Though this should work, I'm wondering if we can do something like
```
str := fmt.Sprintf(""+
	"Name:                      %s\n"+
	"ID:                        %s\n"+
// etc...
)
if state == error {
	str  = fmt.Sprintf("%s",
 		"Provisioning Error Type:   %s\n"+
 		"Provisioning Error Reason: %s\n",
 		str,
 		cluster.Status().ProvisionErrorType(),
 		cluster.Status().ProvisionErrorReason(),
	)
}
fmt.Printf(str)
```

Reply:
@vkareh Yes Updated. Please chk now.